README.md

```diff
@@ -11,16 +11,11 @@ This repo contains all required resources to collect data from Kubernetes cluste
 Detailed instructions are available in our Installation Guides below.
 
 Sumo Logic Helm Chart Version
-|:--------
-[0.17.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.17.0/deploy/README.md)
-[0.16.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.16.0/deploy/README.md)
-[0.15.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.15.0/deploy/README.md)
-[0.14.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.14.0/deploy/README.md)
-[0.13.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.13.0/deploy/README.md)
-[0.12.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.12.0/deploy/README.md)
-[0.11.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.11.0/deploy/README.md)
-[0.10.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.10.0/deploy/README.md)
-[0.9.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.9.0/deploy/README.md)
+| version | status |  
+|--|--|
+|[0.17.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.17.0/deploy/README.md)| current / supported  |
+|[0.16.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.16.0/deploy/README.md) | deprecated |
+|[0.15.0](https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/v0.15.0/deploy/README.md) | deprecated |
```

Comment:
Hmm, this file shouldn't have been changed, it's already in this state on master - might need to update this PR